hooks/badge/useBadges.ts

```diff
@@ -0,0 +1,18 @@
+import { useContractRead } from 'wagmi'
+import henkakuBadge from '@/utils/abis/henkakuBadge.json'
+import { useState } from 'react'
+
+export const useBadges = (contract: string) => {
+  const { data: badges, isError } = useContractRead(
+    {
+      addressOrName: contract,
+      contractInterface: henkakuBadge.abi
+    },
+    'getBadges'
+  )
+
+  return {
+    badges,
```

Comment:
you can use this type definition and provide as badge[]
https://github.com/henkaku-center/omise-interface/pull/90/files#diff-851fc6daa21c2e8eccfa0047119d8734fdcb3358db7403a9851c05b99685fbebR7

Reply:
Answered in the comments above.